charts/unleash/Chart.yaml

```diff
@@ -16,4 +16,4 @@ sources:
   - https://github.com/Unleash/unleash-docker
   - https://github.com/Unleash/helm-charts
 type: application
-version: 1.1.1
+version: 4.0.2
```

Comment:
I would advise against fixing the Helm chart version to the application version, because they are two different things and if we need to fix a minor bug with the chart without updating the app version, this would break.

Reply:
thanks. Agree!